remote_attestation/java/src/com/google/oak/remote_attestation/AttestationStateMachine.java

```diff
@@ -19,40 +19,65 @@
 import com.google.common.base.VerifyException;
 import com.google.oak.remote_attestation.crypto.AeadEncryptor;
 import com.google.oak.remote_attestation.crypto.KeyNegotiator;
+import com.google.protobuf.ByteString;
 import java.security.GeneralSecurityException;
+import java.util.Arrays;
+import oak.remote_attestation.AttestationIdentity;
+import oak.remote_attestation.AttestationInfo;
+import oak.remote_attestation.AttestationReport;
+import oak.remote_attestation.EncryptedData;
 
 public class AttestationStateMachine {
     /**
      * Performs remote attestation and key negotiation.
      */
     public static class Unattested {
+        private static final String test_tee_measurement = "Test TEE measurement";
+        private final byte[] expected_tee_measurement;
         private final KeyNegotiator keyNegotiator;
 
-        public Unattested() throws GeneralSecurityException {
+        public Unattested(byte[] expected_tee_measurement) throws GeneralSecurityException {
+            this.expected_tee_measurement = expected_tee_measurement;
             // Generate client private/public key pair.
             keyNegotiator = new KeyNegotiator();
         }
 
         /** Returns a Diffie-Hellman public key corresponding to the `keyNegotiator`. */
-        public byte[] getPublicKey() throws GeneralSecurityException {
-            return keyNegotiator.getPublicKey();
+        public AttestationIdentity getIdentity() throws GeneralSecurityException {
+            // Generate `AttestationInfo`.
+            AttestationReport report = AttestationReport.newBuilder()
+                    .setMeasurement(ByteString.copyFrom(test_tee_measurement.getBytes()))
+                    .build();
+            AttestationInfo attestationInfo = AttestationInfo.newBuilder()
+                    .setReport(report)
+                    .build();
+
+            AttestationIdentity identity = AttestationIdentity.newBuilder()
+                    .setPublicKey(ByteString.copyFrom(keyNegotiator.getPublicKey()))
+                    .setAttestationInfo(attestationInfo)
+                    .build();
+            return identity;
```

Comment:
I'm confused by the function comment; in what way is this identity a public key?